app/src/main/java/com/keylesspalace/tusky/components/timeline/TimelineTypeMappers.kt

```diff
@@ -207,8 +207,8 @@ fun TimelineStatusWithAccount.toViewData(gson: Gson, isDetailed: Boolean = false
             id = status.serverId,
             url = null, // no url for reblogs
             account = this.reblogAccount!!.toAccount(gson),
-            inReplyToId = null,
-            inReplyToAccountId = null,
+            inReplyToId = status.inReplyToId,
```

Comment:
But the boost itself is not the reply, right? The *actionable* status is the reply

Reply:
What is boosted ("status") can be a reply (too).

This info is "unused".

Reply:
Ah, I see now, this is here as a hack to populate the inreplyto timeline account of the concrete statusviewdata when it's a boost of a reply.
…but as you say, we're not displaying that information, so it could be reverted to avoid confusing the issue (although the status/actionable thing is already confusing)